presto-main/src/main/java/io/prestosql/server/protocol/Query.java

```diff
@@ -874,7 +874,7 @@ private static StageStats toStageStats(StageInfo stageInfo)
                 .build();
     }
 
-    private static Set<String> globalUniqueNodes(StageInfo stageInfo)
+    public static Set<String> globalUniqueNodes(StageInfo stageInfo)
```

Comment:
May be need to get only those where the stage has non-zero spill stats

Reply:
Fixed